package.json

```diff
@@ -1,9 +1,10 @@
 {
   "private": true,
   "dependencies": {
-    "@actions/core": "*",
-    "@actions/exec": "*",
-    "@actions/github": "*"
+    "@actions/cache": "^0.2.1",
+    "@actions/core": "^1.2.0",
+    "@actions/exec": "^1.0.1",
+    "@actions/github": "^1.1.0"
```

Comment:
this downgraded all of the packages -- you'll want to rebase on master and keep the upgraded things